src/pages/UnlockedWallet/AddressesPage/AddressGridRow.tsx

```diff
@@ -59,17 +60,32 @@ const AddressGridRow = ({ addressHash, className }: AddressGridRowProps) => {
   const [isAddressDetailsModalOpen, setIsAddressDetailsModalOpen] = useState(false)
 
   const assetsWithBalance = assets.filter((asset) => asset.balance > 0)
-  const [knownAssets, unknownAssets] = partition(assetsWithBalance, (asset) => !!asset.symbol)
-
-  const [assetSymbolTexts, setAssetSymbolTexts] = useState<string[]>([])
-  const [nbOfHiddenSymbols, setNbOfHiddenSymbols] = useState(0)
 
   const assetsCellRef = useRef<HTMLDivElement>(null)
 
   if (!address) return null
 
   const fiatBalance = calculateAmountWorth(BigInt(address.balance), price ?? 0)
 
+  const renderAssetLogos = () => {
+    const [displayedAssets, hiddenAssets] = chunk(assetsWithBalance, maxDisplayedAssets)
```

Comment:
This way we are only taking the first 2 chunks and we're losing the rest! I'll update the code to fix that

Reply:
Arh, didn't notice that this fn is clever enough to create an infinite number of chunks if needed, and I only tested with a length of less than max * 2. Sorry about that.

Reply:
Haha I saw. I'm okay with that :)